benchalerts/benchalerts/conbench_dataclasses.py

```diff
@@ -0,0 +1,254 @@
+"""Dataclasses for organizing and storing relevant info from Conbench."""
+
+from dataclasses import dataclass
+from typing import List, Optional
+
+from benchclients.logging import fatal_and_log
+
+
+@dataclass
+class BenchmarkResultInfo:
+    """Track and organize specific info about one BenchmarkResult."""
+
+    name: str
+    link: str
+    has_error: bool
+    has_z_regression: bool
+    run_id: str
+    run_reason: str
+    run_time: str
+    run_link: str
```

Comment:
...also from this name I'd assume this is `result.info`. Maybe there's something more descriptive than `info`?

Reply:
Yes! #720 

>Maybe there's something more descriptive than info?

Good callout that `info` is a field of benchmark results already. Honestly I think that's the one that needs to be clarified, not this one 😉 Since this is an internal-only data structure I'm not too worried.